cleaners/brave.xml

```diff
@@ -175,4 +174,9 @@
     <action command="sqlite.vacuum" search="file" path="%LocalAppData%\Google\Chrome\User Data\Default\Plugin Data\Google Gears\localserver.db"/>
     <action command="sqlite.vacuum" search="file" path="%LocalAppData%\Google\Chrome\User Data\Default\Plugin Data\Google Gears\permissions.db"/>
   </option>
+  <option id="site_preferences">
+   <label>Site preferences</label>
+   <description>Shield settings and site permissions</description>
```

Comment:
@ebaldridge 

The string "Settings for individual sites" is already translated (as used in Firefox, Waterfox, and Palemoon). Do you think this would be clear enough?